react-app/src/app/screens/Game/components/Info/layout.js

```diff
@@ -12,3 +15,12 @@ function Info({ status, movesList }) {
 }
 
 export default Info;
+
+Info.propTypes = {
+  status: PropTypes.string,
+  movesList: PropTypes.arrayOf(PropTypes.element).isRequired
```

Comment:
can you avoid `PropTypes.element` here?. What about using `shape`?

Reply:
just think that you are going to receive an object. So it would be `PropTypes.arrayOf(PropTypes.shape(...propTypes))`. Try to find out what is inside `moveList`. this is not neccesary for the approval of this card. Just trying to see how to solve it ;). Let me know how it goes!